src/Components/Picts.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+export const Picts: React.FC = () => {
```

Comment:
This name should be expanded to `Pictures`

Reply:
ur mom